crates/nu-path/src/helpers.rs

```diff
@@ -10,6 +10,42 @@ pub fn config_dir() -> Option<PathBuf> {
     dirs_next::config_dir()
 }
 
+pub fn vendor_completions_dirs() -> Vec<PathBuf> {
+    let vendor_completions_fn = |mut path: PathBuf| {
+        path.push("nushell");
+        path.push("vendor_completions.d");
+        path
+    };
+    let mut dirs = Vec::new();
+
+    // default global directory
+    let global_default = if cfg!(not(target_os = "windows")) {
+        PathBuf::from("/usr/share")
+    } else {
+        PathBuf::from("C:\\ProgramData")
```

Comment:
I think this is more correctly `%SystemDrive%\ProgramData`, where %SystemDrive% is an env var, since we cannot rely on the system drive to always be C:\ on Windows.

Reply:
Does it work if I just put `%SystemDrive% in there? If it's more elaborate I'd appreciate it if someone else could take over that part, as I'm a bit tight on time right now.

Reply:
Skimming the docs I'd say that this should be done using https://microsoft.github.io/windows-docs-rs/doc/windows/Storage/struct.SystemDataPaths.html#method.ProgramData, but that'd be a new dependency, and I have no clue how to use it, and https://kennykerr.ca/rust-getting-started/calling-your-first-windows-api.html makes this seem fairly complicated and it looks like it'd require unsafe code as well.

Reply:
Ah, but digging into how dirs_next is doing it while documenting it in `default_env.nu` gave me a pointer to the `winapi` crate, which allows this in a very much easier way, so I'll go ahead and implement that.

Reply:
I'll mark this bit as resolved, as I've gotten the program data folder using dirs_next/winapi now.